docs/user-guide/configure-zowe-runtime.md

```diff
@@ -748,15 +833,15 @@ You can obtain the _asid_ from the value of `A=asid` when you issue the followin
 
 ## Starting and stopping the Zowe Cross Memory Server on z/OS
 
-The Zowe Cross Memory server is run as a started task from the JCL in the PROCLIB member ZWESIS01. To start this, issue the operator start command through SDSF:
-
+The Cross Memory server is as a started tasks from the JCL in the PROCLIB members ZWESIS01. It supports reusable address spaces and can be started through SDSF with the operator start command with the REUSASID=YES keyword:
```

Comment:
`The Cross Memory server is as a started tasks...` should be `The Cross Memory server is run as a started tasks...`

Reply:
Done